### DISCUSSION TITLE
Is Statamic v2 compatible with PHP 8.1?

### DISCUSSION THREAD

**Discussion by YoBigRube** [statamic/cms#6048] [Q&A] [answered]
Hi,

We've got multiple Statamic v2.x sites set for web host upgrade. Are any of the Statamic v2 versions compatible with PHP 8.1 (would be running on Ubuntu 22.04LTS post-upgrade)? 

If not, do you know if any 2.x versions are compatible with PHP 7.4?

Before anyone recommends "upgrade to v3" - Obvs upgrading to v3 is the ideal and ultimate goal.

Thnx in advance,

**Comment by duncanmcclean** [comment, marked as answer]
The last PHP version Statamic v2 supports is PHP 7.4 (from 2.11.19 forwards).

**Reply by YoBigRube** [reply]
Thanks so much for your help with this @duncanmcclean!! My bad for the delayed response as well. 'ppreciate your help. 

Take care,
